src/boards/board.module.ts

```diff
@@ -0,0 +1,4 @@
+import { Module } from "@nestjs/common";
+
+@Module({})
```

Comment:
module 안에 controller와 service 에 대한 의존도 설정도 필요할 거 같습니당

```suggestion
@Module({})
 controllers: [BoardController],
 providers: [BoardService]
```